components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -119,8 +118,10 @@ - (void)setSelectedViewController:(nullable UIViewController *)selectedViewContr
   }
 
   // Remove current VC and add new one.
-  [self removeContentViewController:self.selectedViewController];
-  [self addContentViewController:selectedViewController];
+  [self.selectedViewController.view removeFromSuperview];
+  [self.content addSubview:selectedViewController.view];
+  [selectedViewController didMoveToParentViewController:self];
```

Comment:
I think `didMoveToParentViewController:` should be called when the view controller is added as a child.

Reply:
The docs suggest that it's called immediately after calling `addSubview:`. So it's called every time the view is added as a subview. I have some honest concerns about how to test VC/view lifecycle for this situation.  If you have any ideas, I'd welcome the suggestions.

Reply:
I think the docs suggest calling it immediately after `addChildViewController:`.

It also appears that this is the behavior of `UITabBarController`.

What are your concerns regarding testing?

> If you are implementing your own container view controller, it must call the didMoveToParentViewController: method of the child view controller after the transition to the new controller is complete or, if there is no transition, immediately after calling the addChildViewController: method.

Reply:
Ok. I might have been too tired reading the docs and UIVC programming guides last night. >_<

Reply:
:) no worries!

Reply:
Done.